packages/cxl-ui/scss/global/cxl-adeft-accordion.scss

```diff
@@ -1,56 +1,50 @@
 @use "~@conversionxl/cxl-lumo-styles/scss/mq";
 @use "../mixins";
 
-$bg-color: #0000ff;
 $standard-margin: 16px;
 $medium-margin: 24px;
 $large-margin: 32px;
 
 cxl-adeft-accordion {
   vaadin-vertical-layout {
     display: flex;
-    border-top: 1px solid $bg-color;
-    border-left: 4px solid $bg-color;
   }
-  --lumo-secondary-text-color: #000;
+
   --lumo-line-height-m: 20px;
   --lumo-line-height-xs: 20px;
```

Comment:
No pixel wrestling, everything should be using or `calc()` with sizing units.

Reply:
done: latest commit